scripts/validate_docstrings.py

```diff
@@ -471,6 +471,43 @@ def parameter_desc(self, param):
                 desc = desc[:desc.index(full_directive)]
         return desc
 
+    def parameter_desc_list(self, param):
+        """Return a parameter description as a list.
+
+        Each line of the parameter docstring is a string in the list.
+        """
+        i = list(self.doc_parameters).index(param)
+        desc_list = [line
+                     for line in self.doc._parsed_data['Parameters'][i][-1]
```

Comment:
Can you maybe add a comment as to what is going on here? Not immediately clear what `i` and and the -1 indexer represent